nameable.rb

```diff
@@ -0,0 +1,6 @@
+class Nameable
+  def correct_name(name)
+    name = name.capitalize
+    name[0, 10] if name.length > 10
+  end
```

Comment:
- [ ] **Required:** The correct name in this class is required to watch out for errors. Consider the code snippet below. 

```
def correct_name
  raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
end
```